ram/wram.asm

```diff
@@ -2868,15 +2868,15 @@ wPlayerGender::
 ;	0 male
 ;	1 female
 	db
-wd473:: ds 1
-wd474:: ds 1
-wd475:: ds 1
-wd476:: ds 1
-wd477:: ds 1
-wd478:: ds 1
+; mobile profile
+wPlayerAge:: ds 1
+wPlayerPrefecture:: ds 1
+wPlayerZipcode:: ds 4
```

Comment:
Japan has [postal codes](https://en.wikipedia.org/wiki/Postal_codes_in_Japan), not ZIP codes.
```suggestion
wPlayerPostalCode:: ds 4
```